docs/developers/reference/api/linea-gettransactionexclusionstatusv1.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: linea_getTransactionExclusionStatusV1
+description: Reference content for the linea_getTransactionExclusionStatusV1 method.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `linea_getTransactionExclusionStatusV1`
+
+Queries a database maintained by the sequencer to check if a transaction was rejected because it 
```

Comment:
should be `by the transaction exclusion API service`, the database was not maintained by the sequencer (which is more acts as the caller to the write method)